lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -1445,6 +1445,127 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
                       binder.op, resultType, operand, constAlpha);
                   return success();
                 });
+  patterns.onOp(
+      "LRN", 13, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
```

Comment:
You can specify the `sinceVersion` to be 1. 1 vs 13 just have the allowed dtype difference.

Reply:
Done.